cmds/core/hostname/hostname_unix.go

```diff
@@ -0,0 +1,20 @@
+// Copyright 2012-2017 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// +build !plan9
+
+// hostname prints or changes the system's hostname.
```

Comment:
also remove this package comment.. hostname.go already has one

Reply:
done